src/components/PlioQuestion.vue

```diff
@@ -41,6 +41,7 @@
                     name="options"
                     v-model="selectedOption"
                     :value="option"
+                    @click="this.selectOption(index)"
```

Comment:
no need to type "this" here
this code will work
`@click="selectOption(index)"`